src/main/java/org/alliancegenome/curation_api/jobs/executors/AlleleExecutor.java

```diff
@@ -19,38 +19,42 @@
 @ApplicationScoped
 public class AlleleExecutor extends LoadFileExecutor {
 
-	@Inject
-	AlleleDAO alleleDAO;
-	@Inject
-	AlleleService alleleService;
+	@Inject AlleleDAO alleleDAO;
+	@Inject AlleleService alleleService;
 
 	public void execLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 
 		BulkManualLoad manual = (BulkManualLoad) bulkLoadFile.getBulkLoad();
 		Log.info("Running with: " + manual.getDataProvider().name());
 
 		IngestDTO ingestDto = readIngestFile(bulkLoadFile, AlleleDTO.class);
-		if (ingestDto == null) return;
-		
+		if (ingestDto == null) {
+			return;
+		}
+
 		List<AlleleDTO> alleles = ingestDto.getAlleleIngestSet();
-		if (alleles == null) alleles = new ArrayList<>();
-		
+		if (alleles == null) {
+			alleles = new ArrayList<>();
+		}
+
 		BackendBulkDataProvider dataProvider = manual.getDataProvider();
-		
+
 		List<Long> alleleIdsLoaded = new ArrayList<>();
 		List<Long> alleleIdsBefore = new ArrayList<>();
 		if (cleanUp) {
 			alleleIdsBefore.addAll(alleleService.getIdsByDataProvider(dataProvider.name()));
 			Log.debug("runLoad: Before: total " + alleleIdsBefore.size());
 		}
-		
+
 		bulkLoadFile.setRecordCount(alleles.size() + bulkLoadFile.getRecordCount());
 		bulkLoadFileDAO.merge(bulkLoadFile);
-		
+
 		BulkLoadFileHistory history = new BulkLoadFileHistory(alleles.size());
 		createHistory(history, bulkLoadFile);
 		boolean success = runLoad(alleleService, history, dataProvider, alleles, alleleIdsLoaded);
-		if(success && cleanUp) runCleanup(alleleService, history, bulkLoadFile, alleleIdsBefore, alleleIdsLoaded);
+		if(success && cleanUp) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.